eng/codeanalysis.base.ruleset

```diff
@@ -207,8 +207,8 @@
     <Rule Id="CA3076" Action="Warning" />       <!-- Insecure XSLT script processing. -->
     <Rule Id="CA3077" Action="Warning" />       <!-- Insecure Processing in API Design, XmlDocument and XmlTextReader -->
     <Rule Id="CA3147" Action="Warning" />       <!-- Mark Verb Handlers With Validate Antiforgery Token -->
-    <Rule Id="CA5350" Action="Warning" />       <!-- Do Not Use Weak Cryptographic Algorithms -->
-    <Rule Id="CA5351" Action="Warning" />       <!-- Do Not Use Broken Cryptographic Algorithms -->
+    <Rule Id="CA5350" Action="Error" />       <!-- Do Not Use Weak Cryptographic Algorithms -->
+    <Rule Id="CA5351" Action="Error" />       <!-- Do Not Use Broken Cryptographic Algorithms -->
```

Comment:
❗ The previous configuration here provides a much better developer experience. Rules should be set to _at most_ Warning, and then turn on TreatWarningsAsErrors in CI only to ensure code with warnings is not merged into the project branches.

Reply:
Thank you for pointing this out.
Adjusted.